src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs

```diff
@@ -12,12 +12,9 @@ public sealed partial class Utf8JsonWriter
         /// <summary>
         /// Writes the pre-encoded property name (as a JSON string) as the first part of a name/value pair of a JSON object.
         /// </summary>
-        /// <param name="propertyName">The JSON encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
-        /// <remarks>
-        /// The property name should already be escaped when the instance of <see cref="JsonEncodedText"/> was created.
-        /// </remarks>
+        /// <param name="propertyName">The name of the property to write.</param>
```

Comment:
We use "The JSON-encoded name of the property to write." in other overloads that accept `JsonEncodedText`

Reply:
Fixed.